iroha_logger/src/lib.rs

```diff
@@ -82,20 +82,16 @@ pub fn init(configuration: &config::LoggerConfiguration) -> Result<(), SetLogger
 
 /// This module contains all configuration related logic.
 pub mod config {
-    use iroha_error::{Result, WrapErr};
+    use iroha_config::derive::Configurable;
     pub use log::LevelFilter;
-    use serde::Deserialize;
-    use std::env;
+    use serde::{Deserialize, Serialize};
 
-    const MAX_LOG_LEVEL: &str = "MAX_LOG_LEVEL";
     const DEFAULT_MAX_LOG_LEVEL: LevelFilter = LevelFilter::Info;
-    const TERMINAL_COLOR_ENABLED: &str = "TERMINAL_COLOR_ENABLED";
     const DEFAULT_TERMINAL_COLOR_ENABLED: bool = false;
-    const DATE_TIME_FORMAT: &str = "DATE_TIME_FORMAT";
     const DEFAULT_DATE_TIME_FORMAT: &str = "%Y-%m-%d %H:%M:%S:%f";
 
     /// Configuration for `Logger`.
-    #[derive(Clone, Deserialize, Debug)]
+    #[derive(Clone, Deserialize, Serialize, Debug, Configurable)]
     #[serde(rename_all = "UPPERCASE")]
     pub struct LoggerConfiguration {
```

Comment:
Don't you need here an attribute `#[config(env_prefix = "LOG_")]`?

Reply:
They were before without prefix, so I left it the same